examples/dms/kafkauser.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: dms.flexibleengine.upbound.io/v1beta1
+kind: KafkaUser
+metadata:
+  annotations:
+    meta.upbound.io/example-id: dms/v1beta1/kafkauser
+  labels:
+    testing.upbound.io/example-name: example_dms_kafka_user1
+  name: example-dms-kafka-user1
```

Comment:
Why example_dms_kafka_user`1` and example-dms-kafka-user`1` ?

Reply:
to avoid any confusion with the user declare in kafka instance